### ISSUE TITLE
Update the `exllamav2` integration

### ISSUE THREAD

**Issue by rlouf** [dottxt-ai/outlines#807]
Similarly to #772 and #782 use logits processors instead of the custom sampling loop.
